pkg/utils/utils.go

```diff
@@ -66,12 +66,19 @@ func GetPfName(pciAddr string) (string, error) {
 			path := filepath.Join(sysBusPci, pciAddr, "net")
 			files, err = ioutil.ReadDir(path)
 			if err == nil {
-				return files[0].Name(), nil
+				if len(files) > 0 {
+					return files[0].Name(), nil
+				}
+				return "", fmt.Errorf("no interface name found for device %s", pciAddr)
 			}
 		}
 		return "", err
+	} else if len(files) > 0 {
+		return files[0].Name(), nil
+	} else {
+		return "", fmt.Errorf("no interface name found for device %s", pciAddr)
```

Comment:
We could skip the final else leaving the last return as the main exit point of the function as in
```
        } else if len(files) > 0 {
		return files[0].Name(), nil
	}
	return "", fmt.Errorf("no interface name found for device %s", pciAddr)
```